scripts/ci/lib.sh

```diff
@@ -306,34 +312,28 @@ push_main_image_set() {
         done
     }
 
-    if [[ "$brand" == "STACKROX_BRANDING" ]]; then
-        local destination_registries=("quay.io/stackrox-io")
-    elif [[ "$brand" == "RHACS_BRANDING" ]]; then
-        local destination_registries=("quay.io/rhacs-eng")
-    else
-        die "$brand is not a supported brand"
-    fi
+    local registry
+    registry="$(registry_from_branding "$brand")"
 
     local tag
     tag="$(make --quiet --no-print-directory tag)"
-    for registry in "${destination_registries[@]}"; do
-        registry_rw_login "$registry"
 
+    registry_rw_login "$registry"
+
+    if is_OPENSHIFT_CI; then
```

Comment:
[nit] TBH, we also don't need the OSCI path here, because we don't build images with OSCI anymore.

Reply:
True. I cleaned those `is_OPENSHIFT_CI` in both `push_main_image_set` and `push_matching_collector_scanner_images` functions.

It looks like the remaining `is_OPENSHIFT_CI` uses in this file are legit.

Please have another look at the PR after these recent changes.